target/linux/realtek/base-files/etc/board.d/02_network

```diff
@@ -36,6 +36,10 @@ hpe,1920-24g)
 	lan_mac_start=$(macaddr_add $lan_mac 2)
 	lan_mac_end=$(macaddr_add $lan_mac $((mac_count2-mac_count1)))
 	;;
+tplink,sg2008p-v1)
+	label_mac=$(mtd_get_mac_binary para 0xfdff4)
+	lan_mac="$label_mac"
```

Comment:
Nothing you have to change here, but I think we should consider implementing nvmem-cells to get this info from DT.